compiler/src/dotty/tools/dotc/core/NameOps.scala

```diff
@@ -27,7 +27,8 @@ object NameOps {
      */
     def apply(s: String)(implicit ctx: Context): String = {
       val marker = "$$$$"
-      val limit: Int = ctx.settings.XmaxClassfileName.value
+
+      val limit: Int = 240
```

Comment:
Could this be lifted to a final val with a descriptive name? Now it is a bit of a magic number

Reply:
`[error] /tmp/1/compiler/src/dotty/tools/dotc/core/NameOps.scala:31:13: expected start of definition
[error]       final val charLimit: Int = 240`

Adding final keyword causes an error. Not sure why

Reply:
Final vals can only appear in fields of classes and objects, not in a method.